datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/operations/GetIndexSizesResolver.java

```diff
@@ -0,0 +1,37 @@
+package com.linkedin.datahub.graphql.resolvers.operations;
+
+import com.linkedin.datahub.graphql.generated.GetIndexSizesResult;
+import com.linkedin.datahub.graphql.generated.IndexSizeResult;
+import com.linkedin.metadata.timeseries.TimeseriesAspectService;
+import com.linkedin.timeseries.TimeseriesIndexSizeResult;
+import graphql.schema.DataFetcher;
+import graphql.schema.DataFetchingEnvironment;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import java.util.stream.Collectors;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+
+@Slf4j
+@RequiredArgsConstructor
+public class GetIndexSizesResolver implements DataFetcher<CompletableFuture<GetIndexSizesResult>> {
```

Comment:
This resolver's endpoints should be behind some policy for security management. Do we want anyone to be able to query for this operational data?